tests/kuttl/tests/galera-network-isolation/02-cleanup.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+- apiVersion: core.openstack.org/v1beta1
+  kind: OpenStackControlPlane
+  name: openstack
```

Comment:
This won't work in combination with the `commands` block below.  The `commands` can run first and then will hang, because the `pvcs` targeted there are being used by the `OpenStackControlPlane`.

Reply:
Actually, I take that back.  I think the problem is just that wrong `name` is used in the `delete` block.  The `OpenStackControlPlane` in this test is called `openstack-network-isolation`.  So `s/openstack/openstack-network-isolation` on L6.